core/src/main/java/org/bitcoinj/wallet/Protos.java

```diff
@@ -16488,6 +16488,10 @@ private Wallet() {
        * <code>ENCRYPTED_ECDH_KEYEXCHANGE_AES = 4;</code>
        */
       ENCRYPTED_ECDH_KEYEXCHANGE_AES(4),
+      /**
+       * <code>ENCRYPTED_AES = 5;</code>
+       */
+      ENCRYPTED_AES(5),
```

Comment:
Fortunately, with this change, we are only adding an enum value.  No changes are made to the wallet format.  

This enum is meant to specify the type of encryption used in wallet, but the values 3-5 are not used for wallet encryption at this time.